sdk/servicebus/ci.yml

```diff
@@ -39,3 +39,10 @@ extends:
       - name: azure-messaging-servicebus
         groupId: com.azure
         safeName: azuremessagingservicebus
+    AdditionalModules:
+        - name: perf-test-core
+          groupId: com.azure
+        - name: azure-messaging-servicebus-track1-perf
+          groupId: com.azure
+        - name: azure-messaging-servicebus-track2-perf
+          groupId: com.azure
```

Comment:
Indentation is wrong, each sub level needs to be 2 spaces to be properly formatted YML